python/sglang/srt/models/deepseek_v2.py

```diff
@@ -1563,13 +1570,22 @@ def post_load_weights(self, is_nextn=False):
                 0, (-1, self_attn.qk_nope_head_dim + self_attn.v_head_dim)
             ).split([self_attn.qk_nope_head_dim, self_attn.v_head_dim], dim=1)
             if not use_deep_gemm_bmm:
-                self_attn.w_kc = w_kc.transpose(1, 2).contiguous().transpose(1, 2)
-                self_attn.w_vc = w_vc.contiguous().transpose(1, 2)
+                if self_attn.w_kc is None:
+                    self_attn.w_kc = w_kc.transpose(1, 2).contiguous().transpose(1, 2)
+                    self_attn.w_vc = w_vc.contiguous().transpose(1, 2)
+                else:
+                    self_attn.w_kc.copy_(
+                        w_kc.transpose(1, 2).contiguous().transpose(1, 2)
+                    )
+                    self_attn.w_vc.copy_(w_vc.contiguous().transpose(1, 2))
                 if (
                     hasattr(self_attn.kv_b_proj, "weight_scale")
                     and self_attn.w_scale is None
                 ):
-                    self_attn.w_scale = self_attn.kv_b_proj.weight_scale
+                    if self_attn.w_scale is None:
+                        self_attn.w_scale = self_attn.kv_b_proj.weight_scale
+                    else:
+                        self_attn.w_scale.copy_(self_attn.kv_b_proj.weight_scale)
```

Comment:
nit: handle these tensor with a new method
```python
def bind_or_assign(target, source):
    if target is not None:
           target.copy_(source)
           return target
    else:
           return source
```
in srt/utils.py

Reply:
fixed.